### PR TITLE
Add changelog for 3.0.0-rc-2

### PR DESCRIPTION
**Pull request by Stexxe**

### CODE REVIEW

**Review by bjhham — APPROVED** [review]

**Comment by bjhham** [comment]
One more small feature I neglected to update in YT:

- [KTOR-7504](https://youtrack.jetbrains.com/issue/KTOR-7504)

**Comment by bjhham** [comment]
Oh, also this one: [KTOR-7427](https://youtrack.jetbrains.com/issue/KTOR-7427) `ByteReadChannel.readFully(dst: ByteArray, offset: Int, length: Int)` is missing since 3.0.0-beta-2

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+* UninitializedPropertyAccessException for `request` property when using MonitoringEvent(RoutingCallStarted) ([KTOR-7449](https://youtrack.jetbrains.com/issue/KTOR-7449))
+* Preconfigured NSURLSession causes requests to hang ([KTOR-7355](https://youtrack.jetbrains.com/issue/KTOR-7355))
+* Failures after upgrading Ktor in test server ([KTOR-7431](https://youtrack.jetbrains.com/issue/KTOR-7431))
+* `InputStream.toByteReadChannel` creates unattached job ([KTOR-7012](https://youtrack.jetbrains.com/issue/KTOR-7012))
+* Maven builds not working ([KTOR-7425](https://youtrack.jetbrains.com/issue/KTOR-7425))
+
+### Improvements
+* Ktor doesn't read System properties from jvm ([KTOR-7492](https://youtrack.jetbrains.com/issue/KTOR-7492))
+* Deprecate `withTestApplication` APIs with ERROR and migrate to `testApplication` ([KTOR-7284](https://youtrack.jetbrains.com/issue/KTOR-7284))
+* Replace `java.time` options in Websockets plugin ([KTOR-7446](https://youtrack.jetbrains.com/issue/KTOR-7446))
```
Comment (osipxd) [inline CHANGELOG.md:23]: Should we highlight breaking changes? These two are breaking

Comment (Stexxe) [reply CHANGELOG.md:23]: @e5l

**Review by osipxd — APPROVED** [review]

**Pull request merged**